src/pylexibank/report.py

```diff
@@ -0,0 +1,193 @@
+import collections
+
+from clldutils.misc import nfilter
+
+from pylexibank.util import get_badge
+
+
+def build_status_badge(dataset):
+    if dataset.dir.joinpath('.travis.yml').exists() and dataset.repo and dataset.repo.github_repo:
+        return "[![Build Status](https://travis-ci.org/{0}.svg?branch=master)]" \
+               "(https://travis-ci.org/{0})".format(dataset.repo.github_repo)
+    return ''
+
+
+def report(dataset, tr_analysis=None, glottolog=None, log=None):
+    #
+    # FIXME: write only summary into README.md
```

Comment:
does this need to actually be fixed? I'm ok with long README files..

Reply:
I don't even recollect whether it was me adding this note ...